autoscale/src/main/java/com/sequenceiq/periscope/controller/AutoScaleClusterV2Controller.java

```diff
@@ -10,6 +10,7 @@
 import com.sequenceiq.periscope.api.model.AutoscaleClusterResponse;
 import com.sequenceiq.periscope.api.model.AutoscaleClusterState;
 import com.sequenceiq.periscope.api.model.StateJson;
+import com.sequenceiq.periscope.converter.ClusterConverter;
```

Comment:
General question: This and AutoScaleClusterV1Controller - Is there some specific reason to move the conversion (domain (or DB) to model (API) from the Service over to the Controller?

Reply:
Yes expect this class all other service api retrieve domain model only. The same domain model can be converted to different types of response objects in the controller, hence it was refactored.